core/types/block_ext_test.go

```diff
@@ -179,72 +146,46 @@ func blockWithNonZeroFields() (*Block, *BlockBodyExtra) {
 	receipts := []*Receipt{{PostState: []byte{11}}}
 
 	block := NewBlock(header, txs, uncles, receipts, stubHasher{})
+	withdrawals := []*ethtypes.Withdrawal{{Index: 12}}
```

Comment:
IIUC you were incrementing to make the values easier to correlate when debugging, so you should bump the ones in the extra.